applications/SwimmingDEMApplication/python_scripts/swimming_DEM_solver.py

```diff
@@ -1,24 +1,25 @@
 from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
 
 # Importing the Kratos Library
-import KratosMultiphysics
+from KratosMultiphysics import *
 from python_solver import PythonSolver
 
 # Import applications
 import KratosMultiphysics.SwimmingDEMApplication as SDEM
 import swimming_DEM_procedures as SDP
+import parameters_tools as PT
 import CFD_DEM_coupling
 import derivative_recovery.derivative_recovery_strategy as derivative_recoverer
 import math
 
 def Say(*args):
-    KratosMultiphysics.Logger.PrintInfo("SwimmingDEM", *args)
-    KratosMultiphysics.Logger.Flush()
+    Logger.PrintInfo("SwimmingDEM", *args)
+    Logger.Flush()
```

Comment:
flushing by default might slow the simulation down in MPI, depending on the cluster